standard_schema/prerelease/HED8.3.0.mediawiki

```diff
@@ -498,7 +505,32 @@ Each term in this vocabulary has a human-readable description and may include ad
 * Data-property <nowiki>{extensionAllowed}[Something that pertains to data or information.]</nowiki>
 ** Data-artifact <nowiki>[An anomalous, interfering, or distorting signal originating from a source other than the item being studied.]</nowiki>
 *** Biological-artifact <nowiki>[A data artifact arising from a biological entity being measured.]</nowiki>
+**** Eye-artifact <nowiki>[Artifact from eyes.]</nowiki> 
+***** Eye-blink-artifact <nowiki>[Artifact from eye blinking.]</nowiki>
+***** Eye-movement-artifact <nowiki>[Artifact from eye movements.]</nowiki>
+****** Horizontal-eye-movement-horizontal-artifact <nowiki>[Artifact from horizontal eye movements.]</nowiki>
+****** Vertical-eye-movement-artifact <nowiki>[Artifact from vertical eye movements.]</nowiki>
+****** Slow-eye-movement-artifact <nowiki>[Artifact from slow eye movements.]</nowiki>
+****** Nystagmus-artifact <nowiki>[Artifact from nystagmus.]</nowiki>
+**** Chewing-artifact <nowiki>[Artifact from chewing.]</nowiki>
+**** Sucking-artifact <nowiki>[Artifact from sucking.]</nowiki>
+**** Tongue-movement-artifact <nowiki>[Artifact from tongue movement (Glossokinetic).]</nowiki>
+**** Rocking-patting-artifact <nowiki>[Artifact from rocking or patting, most commonly seen in infants.]</nowiki>
+**** Movement-artifact <nowiki>[Artifact from motion.]</nowiki>
+**** Respiration-artifact <nowiki>[Artifact from breating.]</nowiki>
+**** Pulse-artifact <nowiki>[Artifact from a pulsating blood vessel near a measurement site, cardiobalistic artfact.]</nowiki>
+**** ECG-artifact <nowiki>[Artifact from pulsation of the heart, timelocked to QRS complex.]</nowiki>
+**** Sweat-artifact <nowiki>[Artifact from sweating, most often seen in EEG.]</nowiki>
+**** EMG-artifact <nowiki>[Artifact from muscle activity at the measurements site.]</nowiki>
 *** Nonbiological-artifact <nowiki>[A data artifact arising from a non-biological source.]</nowiki>
+**** Line-noise-artifact <nowiki>[Power line noise at 50 Hz or 60 Hz.]</nowiki>
+***** <nowiki># {takesValue, valueClass=numericClass, unitClass=frequencyUnits} </nowiki>

```

Comment:
I don't think this should take a placeholder value.  The frequency of the line noise should be annotating separately.  Line noise itself is usually dirty and cannot be isolated to a single frequency.

Reply:
Well, line noise is typically one of the cleanest artifacts in the data, and knowing whether you have to do a notch filter for 60 Hz and harmonics and 50 Hz and harmonics is extremely helpful. Line noise does not occur at other frequencies as far as I know?

Reply:
Okay --- we'll include the placeholder.